lib/reporters/verbose.js

```diff
@@ -74,6 +78,7 @@ VerboseReporter.prototype.finish = function (runStatus) {
 		runStatus.failCount > 0 ?
 			'  ' + colors.error(runStatus.failCount, plur('test', runStatus.failCount), 'failed') :
 			'  ' + colors.pass(runStatus.passCount, plur('test', runStatus.passCount), 'passed'),
+		runStatus.knownFailureCount > 0 ? '  ' + colors.error(runStatus.knownFailureCount, plur('test', runStatus.knownFailureCount), 'known failure') : '',
```

Comment:
This should probably just say "2 known failures". "2 tests known failure" is kinda weird 😄